### ISSUE TITLE
Falsey values for raw output from overridden tags result in the original tag being called

### ISSUE THREAD

**Issue by thibaudcolas** [torchbox/django-pattern-library#16]
In GitLab by @bcdickinson on Jan 26, 2018, 15:51

patterns/organisms/streamfield/story_block.yaml:
```yaml
tags:
  story_block_extra_html_pre:
    value block:
      raw: ''
```

The above gets us the following warning in the logs:
```
No stub data defined for the "story_block_extra_html_pre" tag in the "patterns/organisms/streamfield/story_block.html" template
```

... and the tag errors out because the fake context doesn't provide what it expects.

**Comment by thibaudcolas** [comment]
In GitLab by @bcdickinson on Feb 5, 2018, 09:45

closed via commit 5accf91d59af0bf1c3a9c505d0ea7b993a1a47d8

**Comment by thibaudcolas** [comment]
In GitLab by @bcdickinson on Feb 5, 2018, 09:45

closed via merge request !8
